model_servers/llamacpp_python/tests/conftest.py

```diff
@@ -2,7 +2,7 @@
 import os
 
 MS = pytest_container.Container(
-        url=f"containers-storage:{os.environ['REGISTRY']}/{os.environ['IMAGE_NAME']}",
+        url=f"containers-storage:{os.environ['REGISTRY']}/containers/{os.environ['IMAGE_NAME']}",
```

Comment:
This should instead be done with an env variable. That way if someone is building locally and pushing to their own repo (ex: `quay.io/grpereir/model_server/llamacpp_python`) the tests will still pass.